vm/actor/src/builtin/multisig/mod.rs

```diff
@@ -111,9 +111,8 @@ impl Actor {
             // Return the tx id
             Ok(t_id)
         })??;
-
         // Proposal implicitly includes approval of a transaction
-        Self::approve_transaction(rt, tx_id)?;
+        Self::approve_transaction(rt, tx_id, [0; 32], false)?;
```

Comment:
Not too sure how we want to review this given approve_transaction needs to be updated as the params and functionality have since changed?